libwallet/assets/dcr/wallet.go

```diff
@@ -82,6 +81,7 @@ func initWalletLoader(chainParams *chaincfg.Params, rootdir, walletDbDriver stri
 		ManualTickets:           cfg.ManualTickets,
 		AccountGapLimit:         cfg.AccountGapLimit,
 		MixSplitLimit:           cfg.MixSplitLimit,
+		WatchLast:               20, // Limit number of watched addresses to 20.
```

Comment:
Why this change? Shouldn't be the same as the account gap limit? Especially since a user can generate up to `cfg.AccountGapLimit` unused addresses which will all need to be watched.